source/style-guide/style/code-examples.txt

```diff
@@ -14,6 +14,23 @@ Code Examples
 Use the following guidelines when creating blocks of code as input
 or output examples:
 
+-  Code examples should be formatted and checked to ensure they are idiomatic
```

Comment:
Nitpick:
I think the term `idiomatic` as understood in the context of programming applies to using specific patterns to accomplish common tasks which linters/formatters do not necessarily address. For example, there may be 5 different valid functions to use to open a file, but one method may be preferred by developers.
Instead of idiomatic, you could describe them as following style and formatting guidelines.